src/main/webapp/scripts/toggle.js

```diff
@@ -40,25 +43,55 @@ async function loadHtmlSections(templateUrl, objsUrls, keywordsPromise) {
   const values =  await Promise.all([htmlTemplatePromise, objsPromises]);
   const template = values[0];
   const objs = values[1];
-  return renderTemplateObj(template, objs, keywords);
+  return await renderTemplateObj(template, objs, keywords);
 }
 
-function renderTemplateObj(template, objs, keywords) {
+async function renderTemplateObj(template, objs, keywords) {
   let result = new Object();
   result.keywords = new Array(keywords.length);
   for (let i = 0; i < keywords.length; i++) {
     let term = keywords[i];
-    let keywordsObj = new Object();
-    keywordsObj.term = term;
-    keywordsObj.news = objs[0].articles[term];
-    keywordsObj.actions = objs[1].results[term];
-    result.keywords[i] = keywordsObj;
+    let keywordObj = new Object();
+    keywordObj.term = term;
+    keywordObj.news = objs[0].articles[term];
+    keywordObj.projects = objs[1].results[term];
+    result.keywords[i] = keywordObj;
+  }
 
+  // TODO: Change true to if the user gave their location, and get real coordinates.
+  if (true) {
+    let lat = 33.969058;
+    let lng = -118.422146;
+    let civicObj = await loadObject(`${CIVIC_OBJ_URL}?lat=${lat}&lng=${lng}`);
+    let locationObj = new Object();
+    locationObj.address = civicObj.normalizedInput;
+    locationObj.officials = extractOfficials(civicObj);
+    result.location = locationObj;
   }
   let htmlSections = Mustache.render(template, result);
   return htmlSections;
 }
 
+/**
+ * Extract the government officials from the civic API response.
+ */
+function extractOfficials(civicObj) {
```

Comment:
Is there any reason you're doing this in javascript instead of making a more tailored JSON in the civic servlet? This method seems different than the other methods in toggle.js.

Reply:
I decided to do it in js because it was easier to manipulate due to its structure. However, I do agree that it seems different than the other methods in toggle.js, so I'll add it to a different file.